expo-updates-server/.eslintrc

```diff
@@ -0,0 +1,43 @@
+{
+  "root": true,
+  "extends": [
+    "universe/node",
+    "universe/shared/typescript-analysis"
+  ],
+  "rules": {
+    "handle-callback-err": "off"
+  },
+  "overrides": [
+    {
+      "files": [
+        "*.ts",
+        "*.tsx",
+        "*.d.ts"
+      ],
+      "parserOptions": {
+        "project": "./tsconfig.json"
+      },
+      "rules": {
+        "@typescript-eslint/naming-convention": [
+          "warn",
+          {
+            "selector": "typeLike",
+            "format": [
+              "PascalCase"
+            ]
+          },
+          {
+            "selector": "enumMember",
+            "format": [
+              "UPPER_CASE"
+            ]
+          }
+        ],
```

Comment:
Suggest removing this -- we have this rule to enforce internal standards that line up with the web standard in contrast with TypeScript's convention